spec/jobs/asset_cleanup_job_spec.rb

```diff
@@ -0,0 +1,175 @@
+# frozen_string_literal: true
+
+RSpec.describe AssetCleanupJob do
+  describe "#call" do
```

Comment:
Shouldn't this be `describe "#perform"`?

Reply:
Good catch 👍